iceoryx_hoofs/test/moduletests/test_shared_memory.cpp

```diff
@@ -121,6 +126,8 @@ TEST_F(SharedMemory_Test, CTorWithInvalidArguments)
 TEST_F(SharedMemory_Test, MoveCTorWithValidValues)
 {
     ::testing::Test::RecordProperty("TEST_ID", "2844c9c5-856e-4b51-890d-1418f79f1a80");
+
+    // NOLINTNEXTLINE(cppcoreguidelines-init-variables) initialized before read
```

Comment:
This line is not necessary when line 135 is changed to `int handle = sut->getHandle();`

Reply:
good find